core/src/main/resources/schema/app/20210705075944_DISTX-552_Supporting_custom_service_configs_while_launching_DH_clusters.sql

```diff
@@ -0,0 +1,67 @@
+-- // DISTX-552 Supporting custom service configs while launching DH clusters
+-- Migration SQL that makes the change goes here.
+--  Name: custom_configs_id_seq, Type: Sequence
+CREATE SEQUENCE IF NOT EXISTS custom_configs_id_seq;
+
+ALTER SEQUENCE custom_configs_id_seq OWNER TO postgres;
+
+--  Name: customconfigs, Type: Table, Tablespace:
+CREATE TABLE IF NOT EXISTS customconfigs
+(
+    id                bigint default nextval('custom_configs_id_seq'::regclass)                   not null
+        constraint customconfigs_pk
+            primary key,
+    name varchar(255)                                                                not null,
+    resourcecrn       varchar(255)                                                                not null,
+    created           bigint default (date_part('epoch'::text, now()) * (1000)::double precision) not null,
+    platformversion   text,
```

Comment:
this seems to be overkill for a version. what is expected to be stored here?

Reply:
Here, the corresponding CDH version of the environment will be stored.